site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -58,6 +58,24 @@ You should see output similar to the following:
 {"message":"gameserver update received","severity":"info","time":"2019-10-30T21:46:18.179459+03:00"}
 ```
 
+{{% feature publishVersion="1.38.0" %}}
```

Comment:
```suggestion
```

Reply:
🤔I have removed the feature shortcode in the [latest commit](https://github.com/googleforgames/agones/pull/3560/commits/561f1e4e693fa79a966e4f675311daadc08dadbf_

Reply:
Ah yep, looks like you added the new commit before I finished my review. Looks good!